modules/log-upgrade/6x_creating-and-configuring-a-service-account-for-the-log-collector.adoc

```diff
@@ -0,0 +1,59 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-20
+:_mod-docs-content-type: PROCEDURE
+
+[id="creating-and-configuring-a-service-account-for-the-log-collector_{context}"]
+= Creating and configuring a service account for the log collector
+
+Create a service account for the log collector and assign it the necessary roles and permissions to collect logs.
+
+.Prerequisites
+
+.Procedure
+
+. Create a service account to be used by the log collector:
++
+[source,terminal]
+----
+$ oc create sa logging-collector -n openshift-logging
+----
+
+. Bind the `Cluster Role` role to the service account to be able to write the logs to the Red{nbsp}Hat LokiStack
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user logging-collector-logs-writer -z logging-collector -n openshift-logging
+----
+
+. Assign the necessary permissions to the service account for the collector to be able to collect and forward logs. 
+
+.. Assign permission to collect and forward application logs by running the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z logging-collector -n openshift-logging
+----
+
+.. Assign permission to collect and forward audit logs by running the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z logging-collector -n openshift-logging
+----
+
+.. Assign permission to collect and forward infrastructure logs by running the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-log
+s -z logging-collector -n openshift-logging
```

Comment:
This should be a single line and without the "s":
```
 $ oc adm policy add-cluster-role-to-user collect-infrastructure-log -z logging-collector -n openshift-logging
```

Reply:
fixed